bindata/etcd/pod.gotpl.yaml

```diff
@@ -268,11 +268,14 @@ spec:
           --cert-file /etc/kubernetes/static-pod-certs/secrets/etcd-all-certs/etcd-serving-metrics-NODE_NAME.crt \
           --cacert /etc/kubernetes/static-pod-certs/configmaps/etcd-all-bundles/server-ca-bundle.crt \
           --trusted-ca-file /etc/kubernetes/static-pod-certs/configmaps/etcd-all-bundles/metrics-ca-bundle.crt \
-          --listen-cipher-suites ${ETCD_CIPHER_SUITES}
+        {{- range .EnvVars }}{{ if eq .Name "ETCD_CIPHER_SUITES" }}
```

Comment:
if you want to make this logic a bit easier to test, you can also set a specific variable for the template here:
https://github.com/openshift/cluster-etcd-operator/blob/master/pkg/operator/targetconfigcontroller/targetconfigcontroller.go#L203-L220

makes this a bit more explicit and easier to test from the targetconfigcontroller I think

Reply:
Done.